### PR TITLE
Add install instruction with using add-apt-repository

### PR DESCRIPTION
**Pull request by cornelinux**

### CODE REVIEW

File: doc/installation/ubuntu.rst [hunk 1]
```diff
@@ -55,6 +45,27 @@ Now add the signing key to your system::
...
 
+Now you need to add the repository for your release (either xenial/16.04LTS or bionic/18.04LTS)
+
+You can do this by running the command::
+
+   add-apt-repository http://lancelot.netknights.it/community/xenial/stable
+
+or::
+
+   add-apt-repository http://lancelot.netknights.it/community/bionix/stable
```
Comment (fredreichbier) [inline doc/installation/ubuntu.rst]: *bionic

File: doc/installation/ubuntu.rst [hunk 2]
```diff
@@ -24,17 +24,7 @@ Before installing privacyIDEA 3.0 or upgrading to 3.0 you need to add the reposi
...
-
-   deb http://lancelot.netknights.it/community/xenial/stable xenial main
-
-"xenial" is for Ubuntu 16.04LTS. If you are running Ubuntu 18.04LTS, 
-replase "xenial" with "bionic":
-
-   deb http://lancelot.netknights.it/community/bionic/stable bionic main
-
-Download the signing key::
+The packages are digitially signed. First you need to download the signing key::
```
Comment (fredreichbier) [inline doc/installation/ubuntu.rst]: *digitally

Comment (cornelinux) [reply doc/installation/ubuntu.rst]: done in a0089735 - you are welcome to squash the two commits.

File: doc/installation/ubuntu.rst (hunk 1 above)
Comment (cornelinux) [reply doc/installation/ubuntu.rst]: done in a0089735

**Pull request merged**